### ISSUE TITLE
Missing Visual Focus on My Account: Payment Details

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4332]
Users with disabilities should be able to switch payment methods using the keyboard, currently the options:

- Set as default
- Delete

are not keyboard accessible, therefore not receiving focus, thus no visual focus.

![Screen Shot 2019-08-29 at 8 52 09 PM](https://user-images.githubusercontent.com/15113219/63985798-3f2c7600-ca9f-11e9-80c4-6af049935b54.png)

**Comment by Zeyber** [comment]
Payment detail options can now be focused

**Comment by KateChuen** [comment]
On Chrome, there's the visual focus on the buttons. But, can't select them using the keyboard.

On safari, same as chrome. And the visual focus is weird, the borders are bent.
<img width="1134" alt="Screen Shot 2019-10-21 at 2 58 27 PM" src="https://user-images.githubusercontent.com/45409515/67234647-7a01b900-f413-11e9-9df4-9933b5c41e3f.png">


On Firefox, missing visual focus on the buttons: Delete & Set as Default.

**Comment by KateChuen** [comment]
Tested on dev3.
chrome ✅
firefox ✅
safari ✅
